src/highlighters.cc

```diff
@@ -1082,35 +1082,42 @@ struct ShowWhitespacesHighlighter : Highlighter
 
 struct LineNumbersHighlighter : Highlighter
 {
-    LineNumbersHighlighter(bool relative, bool hl_cursor_line, String separator, int min_digits)
+    LineNumbersHighlighter(bool relative, bool hl_cursor_line, String separator, String separator_cursor, int min_digits)
       : Highlighter{HighlightPass::Move},
         m_relative{relative},
         m_hl_cursor_line{hl_cursor_line},
         m_separator{std::move(separator)},
+        m_separator_cursor{std::move(separator_cursor)},
         m_min_digits{min_digits} {}
 
     static std::unique_ptr<Highlighter> create(HighlighterParameters params, Highlighter*)
     {
         static const ParameterDesc param_desc{
             { { "relative", { false, "" } },
               { "separator", { true, "" } },
+              { "separator-cursor", { true, "" } },
```

Comment:
@mawww what name is do you think appropriate for this? I was thinking that `cursor-separator` works equally well

Reply:
I think both are fine, `-cursor-separator` seems more natural, but `-separator-cursor` seems more consistent with the fact that its kind-of a sub-option of `-separator`.

Reply:
I changed it to `-cursor-separator`, but I'm personally fine with anything. We could also consider "active", "current", or "selected" separator.

Other than that, I consider this to be ready to merge if you decide.